routes/originPyPi.js

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation and others. Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+const asyncMiddleware = require('../middleware/asyncMiddleware')
+const router = require('express').Router()
+const requestPromise = require('request-promise-native')
+
+router.get(
+  '/:namespace?/:name/revisions',
+  asyncMiddleware(async (request, response) => {
+    const { name } = request.params
+    const url = `https://pypi.python.org/pypi/${name}/json`
+    const answer = await requestPromise({ url, method: 'GET', json: true })
+    const result = answer && answer.releases ? Object.keys(answer.releases) : []
```

Comment:
are there any issues with volume here? for example, if there are 1000 releases, does pypi page the results?

Reply:
From my knowledge no this is not a problem. The request returns an object with every single release associated with it. I am yet to run into this case and there is little to no api documentation.

Reply:
I checked a few packages and it looks like it returns all the releases, no pagination.